docs/data/charts/funnel/pyramid.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Charts - Pyramid demonstration
+productId: x-charts
+components: FunnelChart, FunnelPlot
+---
+
+# Charts - Pyramid demonstration
+
+<p class="description">This page demonstrates the pyramid chart.</p>
+
+## Pyramid Chart
+
+The pyramid chart is a variation of the funnel chart.
+
+To create a pyramid chart, set the `curve` property to `pyramid` in the series.
+
+{{"demo": "PyramidFunnel.js"}}
```

Comment:
We could add more demos, like upside down, and mention other features and link to the funnel chart as well.